bundles/org.openhab.binding.siemenshvac/src/main/resources/OH-INF/thing/ozw672.xml

```diff
@@ -0,0 +1,100 @@
+					forget the trailing '/'</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="userName" type="text">
+				<description>User name of the Siemens Hvac gateway</description>
+				<required>false</required>
+				<label>User Name</label>
+				<default>Administrator</default>
+			</parameter>
+			<parameter name="userPassword" type="text">
+				<description>User password of the Siemens Hvac gateway</description>
+				<required>false</required>
+				<label>User Password</label>
+				<default>password</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<channel-type id="outside-temp">
+		<item-type>Number:Temperature</item-type>
+		<label>Outside Temperature</label>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%" step="0.1"/>
+	</channel-type>
+
+	<channel-type id="setpoint-heat-pump">
+		<item-type>Number:Temperature</item-type>
+		<label>Setpoint heat pump</label>
+		<description>Setpoint Heat Pump</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%" step="0.1"/>
+	</channel-type>
+
+	<channel-type id="room-temp-comfort-setpoint-hc">
+		<item-type>Number:Temperature</item-type>
+		<label>RoomTemp ComfSetp HC</label>
+		<description>Room temperature Comfort setpoint HC</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%" step="0.5"/>
+	</channel-type>
+
+	<channel-type id="room-temp-reduce-setpoint-hc">
+		<item-type>Number:Temperature</item-type>
+		<label>RmTmp RedSetp HC</label>
+		<description>Room temp reduced setpoint heat circuit</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%" step="0.5"/>
+	</channel-type>
+
+	<channel-type id="operating-mode-hc">
+		<item-type>Number</item-type>
+		<label>OptgMode HC</label>
+		<description>Operating mode heat circuit</description>
+		<state>
+			<options>
+				<option value="0">Protection mode</option>
+				<option value="1">Automatic mode</option>
+				<option value="2">Reduced mode</option>
+				<option value="3">Comfort mode</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="status-hc2">
+		<item-type>Number</item-type>
+		<label>Status</label>
+		<description>Status heating circuit</description>
+		<category>Temperature</category>
+		<state>
+			<options>
+				<option value="0">---</option>
+				<option value="1">SLT cut out</option>
+				<option value="2">Fault</option>
+				<option value="3">Switch cut out</option>
+				<option value="4">Manual operation active</option>
+				<option value="5">Chimney sweep function, full load</option>
+				<option value="6">Chimney sweep function, partial load</option>
+				<option value="7">Chimney sweep function active</option>
+				<option value="8">Locked, manually</option>
+				<option value="9">Locked, automatically</option>
+			</options>
+		</state>
+	</channel-type>
```

Comment:
In first review, someone tell me that user should be able to create the things manually, and so that I should put some examples of channel inside the binding .xml files.

So thats why it is there, but clearly it's redundant with metadata auto discovery, as the binding will create dynamically the same channel-type in the ChannelTypeProvider.

Let me know, I can just remove them if you thing it's unnecessary.

Reply:
If they are not used, they should be removed. If the channel types are created dynamically by the binding, I assume it's still possible to create Things manually?

Reply:
Ok, I removed them